src/ElectricPokemon.java

```diff
@@ -0,0 +1,52 @@
+public class ElectricPokemon extends Pokemon {
+    int volt;
```

Comment:
Het is verstandig om je klasse variabelen private te maken, anders is het ook niet nodig om getters en setters te maken. JE kunt dan gewoon `electricPokemon.volt` uitvoeren.